### DISCUSSION TITLE
Possiblility to enable the extension for a specific website only (globaly disabled)

### DISCUSSION THREAD

**Discussion by Raziel-SSJ** [uBlockOrigin/uBOL-home#91] [Ideas]
Hi.
This extension is great for blocking YT Ads, but also block/break some websites that don't even show ads in the meantime _(like `1fichier.com`)_
Could it be possible for the extension to be globally disabled and enable it only for a specific website?

**Comment by gorhill** [comment]
Just lower the filtering mode on `1fichier.com`?

What is broken on that site? Ideally you tell us what is broken then we fix in filter lists.

**Reply by Raziel-SSJ** [reply]
> Just lower the filtering mode on `1fichier.com`?

Sorry but this is not the question.

> What is broken on that site? Ideally you tell us what is broken then we fix in filter lists.

Sorry again but I don't think this is the right place for fix request. This is why a new Issue for this problem has already been opened in the appropriate section.
But to answer your question, here is the evidence:
# Without uBOL
### Before log-in
![image](https://github.com/uBlockOrigin/uBOL-home/assets/55009486/7c8815d9-9c8f-49da-9286-88322caa4db0)

### After logged-in
![image](https://github.com/uBlockOrigin/uBOL-home/assets/55009486/292e2901-4a39-4f6e-bea0-73893bdd3341)

# With uBOL
## No cache loaded
### Before log-in
![image](https://github.com/uBlockOrigin/uBOL-home/assets/55009486/54f85be5-a3d7-4e77-88dc-904259453da9)

### After logged-in
![image](https://github.com/uBlockOrigin/uBOL-home/assets/55009486/ebacb1b9-11bb-4724-90c6-0b0451b58a58)

# With uBOL
## Cache loaded  (cache loaded without uBOL)
### After logged-in
![image](https://github.com/uBlockOrigin/uBOL-home/assets/55009486/79c6753c-8681-4b5a-a26d-47f93ec32b8b)

So, almost everything is broken.

**Reply by gorhill** [reply]
It's a Firefox issue, their DNR API is not fully implemented, it does not yet support `third-party` filter option: <https://bugzilla.mozilla.org/show_bug.cgi?id=1797408>, and this is causing the issue here.

You will have to lower the filtering mode to "no filtering" for that site, until the issue is fixed in Firefox.

**Reply by Raziel-SSJ** [reply]
>You will have to lower the filtering mode to "no filtering" for that site, until the issue is fixed in Firefox.

Thank you for the answer but I've already found this workaround.

Still, I think it could be a good addition for people who only want to block ads from a specific website.
Also, since this is not the only (annoying) breakage and this could still appear in future FF updates, I think this could be a good option on this as well.

**Comment by gorhill** [comment]
This can be accomplished using browser permissions.
